.github/workflows/test-asan.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
     paths-ignore:
+      - '.github/*'
+      - !'.github/workflows/test-asan.yml'
```

Comment:
I thought `!` only works with `paths` and not `paths-ignore`. https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#excluding-paths

Reply:
I'm not sure, but they certainly seem to discourage using `!` with `paths-ignore`, so I guess I'll refactor this...